src/rust/catmem/queue.rs

```diff
@@ -109,7 +102,6 @@ impl SharedCatmemQueue {
             }
         }
         self.ring.prepare_closed()?;
-        self.cancel_pending_ops(Fail::new(libc::ECANCELED, "this queue was closed"));
```

Comment:
Where is cancelletaion happening now? I missed this :)

Reply:
This now happens at a centralized place in this function: https://github.com/microsoft/demikernel/blob/2fa835968cee1db9025a5e41efe2bc80236f19a1/src/rust/runtime/mod.rs#L228.

This change went in earlier in another PR. :)

Reply:
Yes, that is what I was thinking. Thank you for the reference :)